sandbox.sh

```diff
@@ -141,7 +141,30 @@ How do you wish to proceed? [1]"
   cmd="rsync -av --no-p --no-times --exclude-from='.sandbox-ignore' --exclude=$ignore_string ./ $SANDBOX_USER@$SANDBOX_LOCATION:$SANDBOX_PUBLIC_THEMES_FOLDER/"
   eval $cmd
 
+elif [[ $1 == "export" ]]; then
+  # Export the site content from your sandbox to a local file
+  export_command='ssh $SANDBOX_USER@$SANDBOX_LOCATION "mkdir /home/wpcom/export ; rm /home/wpcom/export/* ; /home/wpcom/public_html/bin/wp export --dir=/home/wpcom/export"'
```

Comment:
So this command assumes I have some exported content in `/home/wpcom/export already`? Is there a way I could specify a specific site of mine to export from?

Reply:
No, that command will create the file in the /home/wpcom/export folder and then copy it locally.

It's a pretty hacky way to do it I think but I couldn't quite get things to work in another way.  And Bash always just seems like a giant roll of duct tape to me anyway...

It would be a simple change to target a particular site defined in the command execution, yea.

Reply:
Truth be told I have NO idea where there's a `mkdir` followed by an immediate `rm` so that's probably something that should change (even if it doesn't not work)